appcheck/firebase-appcheck-debug/CHANGELOG.md

```diff
@@ -0,0 +1,37 @@
+# Unreleased
+
+# 16.0.1
+* [changed] Updated dependency of `play-services-basement` to its latest
+  version (v18.1.0).
+
+# 16.0.0
+* [changed] [app_check] has exited beta and is now generally available for
+  use.
+
+# 16.0.0-beta06
+* [fixed] Fixed a bug in the [app_check] token refresh flow when using a
+  custom provider.
+
+# 16.0.0-beta05
+* [changed] Internal improvements.
+
+# 16.0.0-beta04
+* [changed] Improved error handling logic by minimizing the amount of requests
+  that are unlikely to succeed.
+
+* [fixed] Fixed heartbeat reporting.
+
+# 16.0.0-beta03
+* [changed] Added `X-Android-Package` and `X-Android-Cert` request headers to
+  [app_check] network calls.
+
+# 16.0.0-beta02
+* [feature] Added [`getAppCheckToken()`](/docs/reference/android/com/google/firebase/appcheck/FirebaseAppCheck#getAppCheckToken(boolean)),
+  [`AppCheckTokenListener`](/docs/reference/android/com/google/firebase/appcheck/FirebaseAppCheck.AppCheckListener),
```

Comment:
Do we expect teams to write in this style? might need Rachel to review at the time of commit?

Reply:
Yes. We already expect teams to do this, although most of the times the link is reviewed/added by Rachel during review.